torch/_dynamo/trace_rules.py

```diff
@@ -35,40 +44,121 @@
 and trace_rules.lookup consolidation is done. Then the explicit listing of skip/inline objects have
 a higher priority, which can be used to override the skipfiles.check rules in some cases.
 """
-torch_name_rule_map = {
+manual_torch_name_rule_map = {
+    "torch.profiler.profiler.profile": TorchCtxManagerClassVariable,
+    "torch.autograd.profiler.profile": TorchCtxManagerClassVariable,
+    "torch.autograd.profiler.record_function": TorchCtxManagerClassVariable,
+    "torch.default_generator#get_state": TorchInGraphFunctionVariable,
+    "torch._C.Generator#get_state": TorchInGraphFunctionVariable,
+    "torch.default_generator#set_state": TorchInGraphFunctionVariable,
+    "torch._C.Generator#set_state": TorchInGraphFunctionVariable,
+    "torch.onnx.is_in_onnx_export": TorchInGraphFunctionVariable,
+    "torch.onnx.operators.shape_as_tensor": TorchInGraphFunctionVariable,
+    "torch.overrides.is_tensor_like": TorchInGraphFunctionVariable,
+    "torch.jit.is_scripting": TorchInGraphFunctionVariable,
+    "torch.jit.is_tracing": TorchInGraphFunctionVariable,
+    "torch.jit.annotate": TorchInGraphFunctionVariable,
+    "torch.distributed.is_available": TorchInGraphFunctionVariable,
+    "torch.distributed.is_initialized": TorchInGraphFunctionVariable,
+    "torch.distributed.get_rank": TorchInGraphFunctionVariable,
+    "torch.distributed.get_world_size": TorchInGraphFunctionVariable,
+    "torch.distributed._tensor.DTensor#from_local": TorchInGraphFunctionVariable,
+    "torch._utils.is_compiling": TorchInGraphFunctionVariable,
+    "torch.overrides.get_default_nowrap_functions": TorchInGraphFunctionVariable,
+    "torch.fx._symbolic_trace.is_fx_tracing": TorchInGraphFunctionVariable,
+    "torch._dynamo.external_utils.is_compiling": TorchInGraphFunctionVariable,
+    "torch.autograd.graph.disable_saved_tensors_hooks": TorchInGraphFunctionVariable,
+}
+
+
+auto_torch_name_rule_map = {
```

Comment:
I don't know what "auto" here means

Reply:
```auto``` means this is generated by the heuristic function ```gen_allowed_objs_and_ids```, ```manual``` means this function is added in ad hoc way.

Reply:
It's just weird to me that there are entries in the auto list lol

Reply:
Ah, it just means this list is generated automatically, I'm open to better name if you have.